Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -513,14 +508,12 @@ public void indexConcepts(String userName)
                                 // performed using the BooleanQuery q2.
                                 shortTermBooleanQueryBuilder.add(q1, BooleanClause.Occur.SHOULD);
                             }
-
                         }
```

Comment:
can't you merge the first if and the else block by always creating the shortTermsQueryBuilder, but only if it's also shortPhrase searchable create q2? Would make much cleaner code.